### PR TITLE
Snagging: Profile & Schedule

### PR DESCRIPTION
**Pull request by alexxpetrov**
Closes:
- https://github.com/sparkletown/internal-sparkle-issues/issues/1831
- https://github.com/sparkletown/internal-sparkle-issues/issues/1829 ( partial )
- https://github.com/sparkletown/internal-sparkle-issues/issues/1845 ( Only p0 )

* Fixed profile icons
* Profile name validation

### CODE REVIEW

File: src/components/attendee/ProfileOverlay/ProfileOverlay.tsx [hunk 1]
```diff
@@ -106,7 +106,7 @@ export const ProfileOverlay: React.FC<ProfileOverlayProps> = ({
       }, 2000);
     }
   }, [isSuccess]);
-
+  console.log(values);
```
Comment (colinhowe) [inline src/components/attendee/ProfileOverlay/ProfileOverlay.tsx]: Delete plz

File: src/components/attendee/ScheduleOverlay/ScheduleEvents.tsx [hunk 2]
```diff
@@ -63,7 +63,7 @@ export const ScheduleEvents: React.FC<ScheduleEventsProps> = ({
     const daysEvents = liveAndFutureEvents.filter(
       isEventWithinDateAndNotFinished(day)
     );
-
+    console.log(liveAndFutureEvents, daysEvents, day);
```
Comment (colinhowe) [inline src/components/attendee/ScheduleOverlay/ScheduleEvents.tsx]: Delete

**Review by colinhowe — APPROVED** [review]
Some console logs to remove and then merge

File: src/components/attendee/ScheduleOverlay/ScheduleEvents.tsx (hunk 2 above)
Comment (alexxpetrov) [reply src/components/attendee/ScheduleOverlay/ScheduleEvents.tsx]: [dbc88e0](https://github.com/sparkletown/sparkle/pull/3044/commits/dbc88e0c84bb6ea6dd9f509052670b11cf5067e5)

File: src/components/attendee/ProfileOverlay/ProfileOverlay.tsx (hunk 1 above)
Comment (alexxpetrov) [reply src/components/attendee/ProfileOverlay/ProfileOverlay.tsx]: [dbc88e0](https://github.com/sparkletown/sparkle/pull/3044/commits/dbc88e0c84bb6ea6dd9f509052670b11cf5067e5)

**Pull request merged**